src/include/snapper/dialogs.rb

```diff
@@ -25,12 +25,9 @@
 # Authors:	Jiri Suchomel <jsuchome@suse.cz>
 
 module Yast
-
```

Comment:
it is just matter of style and rubocop goal is to have unified style across files. It is similar question if there should be space after `(` or not. For me empty lines make sense in code where it provide additional separation of logic parts of method, but empty line after module definition do not provide additional information.
According to ruby style guide empty lines are used for separation of logic part and separation of individual methods - https://github.com/bbatsov/ruby-style-guide#empty-lines-between-methods
I am fine if we agree that we want to have space after module or class definition, but we should be consistent, so modify shared yast style guide.